src/pconn.cc

```diff
@@ -584,14 +583,14 @@ PconnModule::remove(PconnPool *aPool)
 void
 PconnModule::dump(StoreEntry *e)
 {
-    typedef Pools::const_iterator PCI;
-    int i = 0; // TODO: Why number pools if they all have names?
-    for (PCI p = pools.begin(); p != pools.end(); ++p, ++i) {
+    PackableStream yaml(*e);
```

Comment:
Please move this conversion one level up, into DumpWrapper() implementation. It will reduce code changes when all cache manager reports become stream-based.

Rule of thumb: Start low/deep but then go as "high" in the call chain as possible without heroic efforts (i.e. stop where you would be forced to make large and/or out-of-scope changes).

Reply:
done